lib/Tags/TagsForm.js

```diff
@@ -96,21 +96,17 @@ const TagsForm = ({
   return (
     <FormattedMessage id="stripes-smart-components.enterATag">
       {placeholder => (
-        <FormattedMessage id="stripes-smart-components.tagsTextArea">
-          {ariaLabel => (
-            <MultiSelection
-              id="acqtags"
-              placeholder={placeholder}
-              aria-label={ariaLabel}
```

Comment:
there is no such prop in MultiSelection, IIUC. So you should pass `label` or `aria-labelledby`